### PR TITLE
[TRUNK-5242] Remove duplication of Logger

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/TRUNK-5242

Deleted duplicated declaration of Logger variable, this class should have only one field Logger which should be private static final.

### CODE REVIEW

**Comment by dkayiwa** [comment]
@BartlomiejRasztabiga there are still more in this same class. Just search for org.slf4j.Logger

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Done :)

File: web/src/main/java/org/openmrs/web/Listener.java [hunk 1]
```diff
@@ -172,8 +166,7 @@ public void contextInitialized(ServletContextEvent event) {
 				
 				//ensure that we always log the runtime properties file that we are using
 				//since openmrs is just booting, the log levels are not yet set. TRUNK-4835
-				Logger.getLogger(getClass()).setLevel(Level.INFO);
```
Comment (dkayiwa) [inline web/src/main/java/org/openmrs/web/Listener.java]: How about setting the log level?

Comment (BartlomiejRasztabiga) [reply web/src/main/java/org/openmrs/web/Listener.java]: I think there is no setLevel in slf4j, it is replaced with info() method call

Comment (dkayiwa) [reply web/src/main/java/org/openmrs/web/Listener.java]: So how are log levels set?

Comment (BartlomiejRasztabiga) [reply web/src/main/java/org/openmrs/web/Listener.java]: I think slf4j doesn't allow dynamic level changes. Maybe it depends on implementation

Comment (dkayiwa) [reply web/src/main/java/org/openmrs/web/Listener.java]: Can we then revert to the original one which allowed setting the log level?

Comment (BartlomiejRasztabiga) [reply web/src/main/java/org/openmrs/web/Listener.java]: @dkayiwa  Fixed :)

Comment (dkayiwa) [reply web/src/main/java/org/openmrs/web/Listener.java]: Ticket status says you are still doing more work on this. 😊

Comment (BartlomiejRasztabiga) [reply web/src/main/java/org/openmrs/web/Listener.java]: Done :)

**Pull request merged**